flow/legacy.py

```diff
@@ -29,3 +31,41 @@ def __iter__(self):
 
     def __len__(self):
         return len(self._find_ids())
+
+
+class JobOperation(JobsOperation):
```

Comment:
Unfortunately this class was exposed in the root namespace, so we need to deprecate it. But we can be quick about it, since I do not consider this class core API.